pkg/oci/load_balancer.go

```diff
@@ -436,11 +436,17 @@ func (cp *CloudProvider) updateBackendSet(ctx context.Context, lbOCID string, ac
 
 		workRequestID, err = cp.client.LoadBalancer().CreateBackendSet(ctx, lbOCID, action.Name(), bs)
 	case Update:
-		err = cp.securityListManager.Update(ctx, lbSubnets, nodeSubnets, sourceCIDRs, listenerPort, backendPort, healthCheckPort)
-		if err != nil {
+		// FIXME(apryde): This is inelegant and inefficient. Update() should be refactored
```

Comment:
Create ticket for this one so we can track it.

Reply:
https://github.com/oracle/oci-cloud-controller-manager/issues/152